nni/retiarii/experiment/pytorch.py

```diff
@@ -240,7 +242,12 @@ def _check_exp_status(self) -> bool:
         try:
             while True:
                 time.sleep(10)
-                status = self.get_status()
+                # this if is to deal with the situation that
```

Comment:
Better to put this logic into a member function?

And I'm not sure whether this is correct? @liuzhe-lz

Reply:
leave the refactor to the next release